GameEngine/Engine.cs

```diff
@@ -44,7 +45,39 @@ public static void Start(Func<IGameData> gameDataFactory, int loadFromSlot = 0)
                 {
                     // Otherwise if we're not starting a new game, show the game intro text
                     // Show the game introduction
-                    engineInstance.StartNewGame();
+                    int displaycount = 1;
+                    Dictionary<int, string> SlotNames = new Dictionary<int, string>();
+                    Console.WriteLine("Hello, wellcome to our game, to load or start a new game, pick a number.");
+                    foreach(var slotName in GameState.GetValidSaveSlotNames())
+                    {
+                        Console.WriteLine();
+                        Console.WriteLine($"{displaycount}. {slotName}");
+                        SlotNames.Add(displaycount, slotName);
+                        displaycount++;
+                    }
+                    Console.WriteLine();
+                    Console.WriteLine($"{displaycount}. Start new Game");
+                    Console.WriteLine("-----------------------------------------");
+                    Console.Write("> ");
+                    int input = int.Parse(Console.ReadLine());
+                    if (SlotNames.ContainsKey(input))
+                    {
+                        string slotToFind;
+                        slotToFind = SlotNames[input];
+                        Console.WriteLine($"Loading {slotToFind}.......");                 
+                        Console.ReadLine();
+                        Console.WriteLine($"Completed Loading {slotToFind}.");
+                        Console.ReadLine();
+                        GameState.LoadGameState(slotToFind);
```

Comment:
Doesn't the LoadGameState return the loaded game state ? It doesn't look like you are doing anything with that here.

Reply:
i need help with what you are asking here.